help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,142 @@
+---
+keywords: crm;CRM;crm destinations;salesforce crm;salesforce crm destination
+title: Salesforce CRM connection
+description: The Salesforce CRM destination allows you to export your account data and activate it within Salesforce CRM for your business needs.
+---
+
+# [!DNL Salesforce CRM] connection
+
+## Overview {#overview}
+
+[Salesforce CRM](https://www.salesforce.com/) is a popular Customer Relationship Management (CRM) platform.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_composite_upsert_example.htm?q=contacts), which allows you to add Accounts and create Contacts after activating them within a new Salesforce segment for your business needs.
```

Comment:
what is the reason for using uppercase Accounts and Contacts?

Reply:
Was meant to refer to Salesforce entities but you are correct its not related to AEP have changed to lowercase.